docs/functions.rst

```diff
@@ -1332,6 +1332,58 @@ Example:
           snapshotClass: do-block-storage
 
 
+CreateCSISnapshotStatic
+-----------------------
+
+This function creates a pair of CSI VolumeSnapshot and VolumeSnapshotContent.
+It is commonly used to pre-provision snapshot volume and snapshot volume content
+resources as described in the Kubernetes `documentation
+<https://kubernetes.io/docs/concepts/storage/volume-snapshots/#static>`_.
+
+The deletion behavior is defined by the ``deletionPolicy`` property (``Retain``,
+``Delete``) of the snapshot class.
+
+Arguments:
+
+.. csv-table::
+   :header: "Argument", "Required", "Type", "Description"
+   :align: left
+   :widths: 5,5,5,15
+
+   `name`, Yes, `string`, name of the new VolumeSnapshot
+   `namespace`, Yes, `string`, namespace of the new VolumeSnapshot
+   `driver`, Yes, `string`, name of the CSI driver for the new VolumeSnapshotContent
+   `handle`, Yes, `string`, unique identifier of the volume created on the storage backend used as the source of the new VolumeSnapshotContent
+   `snapshotClass`, Yes, `string`, name of the VolumeSnapshotClass to use
+
+Outputs:
+
+.. csv-table::
+   :header: "Output", "Type", "Description"
+   :align: left
+   :widths: 5,5,15
+
+   `name`,`string`, name of the new VolumeSnapshot
+   `namespace`, string, namespace of the new VolumeSnapshot
+   `restoreSize`, string, required memory size to restore the volume
+   `snapshotContent`, string, name of the new VolumeSnapshotContent
```

Comment:
I think we should be explicit that we are talking about VolumeSnapshot Kubernetes resources here. Because it sounds like we are talking about actual volume snapshots. WDYT.